doc/script/collect_tex.py

```diff
@@ -0,0 +1,63 @@
+
+# Copyright 2020, Battelle Energy Alliance, LLC
+# ALL RIGHTS RESERVED
+"""
+  Constructs documentation files for HERON.
+"""
+import os
+import sys
+
+teal_path = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', 'src'))
+build_path = os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'build'))
+src_path = os.path.abspath(os.path.join(build_path, '..', 'src'))
```

Comment:
afaik TEAL uses the `camelBack` convention for variable naming, so everything using `snake_case` will need to be converted. I fought a small war in HERON to get `snake_case` included, and tried to do that for TEAL too but did not succeed unfortunately.